src/main/java/org/sagebionetworks/bridge/udd/helper/SesHelper.java

```diff
@@ -25,6 +25,7 @@
     // TODO: move these to config
     private static final String DEFAULT_FROM_ADDRESS = "Bridge (Sage Bionetworks) <support@sagebridge.org>";
     private static final String SUBJECT_TEMPLATE = "Your requested data from %s";
+
     private static final String BODY_TEMPLATE_HTML = "<html>%n" +
             "   <body>%n" +
             "       <p>To download your requested data, please click <a href=\"%s\">here</a>.</p>%n" +
```

Comment:
<p>Your <a href="blahblahblah">requested data download</a> is now available.</p> ?